content/blog/2020-03-tibble-3-0-0/index.Rmarkdown

```diff
@@ -0,0 +1,185 @@
+---
+title: tibble 3.0.0
+slug: tibble-3-0-0
+description: >
+    tibble 3.0.0 is on CRAN now! Tibbles are a modern reimagining of the data frame, keeping what time has shown to be effective, and throwing out what is not, with nicer default output too! This article describes the latest major release and provides an outlook on further developments
+date: 2020-03-30
+author: Kirill Müller
+photo:
+  url: https://unsplash.com/photos/dbOV1qSiL-c
+  author: Vinicius Amano
+categories:
+- package
+- programming
+---
+
+
+```{r setup, include = FALSE}
+library(tidyverse)
+knitr::opts_chunk$set(collapse = TRUE, comment = "#>", error = TRUE)
+```
+
+
+Version 3.0.0 of the tibble package is on CRAN now. Tibbles are a modern reimagining of the data frame, keeping what time has shown to be effective, and throwing out what is not, with nicer default output too! Grab the latest version with:
+
+```r
+install.packages("tibble")
+```
+
+Tibble now fully embraces vctrs, using it under the hood for its subsetting and subset assignment ("subassignment") operations.
+Accessing and updating rows and columns is now based on a rock-solid framework and works consistently for all types of columns, including list, data frame, and 
+matrix columns.
+We believe that the changes will ultimately lead to better and safer code.
```

Comment:
I'd indicate that the rest of the post is about the precise technical definition of a tibble, and therefore mostly suited for interested R programmers.